server/package.json

```diff
@@ -8,7 +8,6 @@
   },
   "dependencies": {
     "bcryptjs": "^2.4.3",
-    "graphql": "^14.5.3",
```

Comment:
Apparently we have to respect the exact version used by yoga, else we get different realm errors